deep_regression.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import absolute_import, division, print_function
+
+import tensorflow as tf
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.io as sio
+import pandas as pd
+from sklearn.utils import shuffle
+from sklearn.model_selection import train_test_split
+import tensorflow.contrib.slim as slim
+from tensorflow.contrib.framework.python.ops.variables \
+    import get_or_create_global_step
+from tensorflow.python.platform import tf_logging as logging
+
+
+'''
+Changeable parameters OR variables
```

Comment:
We don't need this comment. If you want the user to be able to set these parameters, make this a class and set these in the constructor.